exec.go

```diff
@@ -46,14 +55,43 @@ func (exec *EXEC) Exports() modules.Exports {
 }
 
 // Command is a wrapper for Go exec.Command
-func (*EXEC) Command(name string, args []string, option CommandOptions) string {
+func (*EXEC) Command2(name string, args []string, option CommandOptions) CommandReturn {
 	cmd := exec.Command(name, args...)
+
+	var outb, errb bytes.Buffer
+	cmd.Stdout = &outb
+	cmd.Stderr = &errb
+
 	if option.Dir != "" {
 		cmd.Dir = option.Dir
 	}
-	out, err := cmd.Output()
+
+	if len(option.Env) > 0 {
+		cmd.Env = append(cmd.Environ(), option.Env...)
+	}
+	err := cmd.Run()
+	statusCode := cmd.ProcessState.ExitCode()
 	if err != nil {
-		log.Fatal(err.Error() + " on command: " + name + " " + strings.Join(args, " "))
+		// Keep default behaviour backwards compatible
+		if option.FatalError == nil || *option.FatalError {
+			log.Fatal(err.Error() + " on command: " + name + " " + strings.Join(args, " "))
+		} else {
+			log.Print(err.Error() + " on command: " + name + " " + strings.Join(args, " "))
+		}
+	}
+
+	return CommandReturn{
```

Comment:
Maybe we should add the error from the execution of the command, as we are now just printing it but there's no way to check if the command was actually executed or there was an error.

Reply:
I didn't find it clear in the documentation, but from what I understood, When the command actually finished, there will be a exitCode >= 0

When the comand is not executed, the exitCode will be < 0 (or simply -1 from my short experience).

If clearly documented, I think this should be enougth

Reply:
Also, the examples  could be a better test case. I will also do that.